doc/classes/Particles.xml

```diff
@@ -15,13 +15,14 @@
 			<return type="AABB">
 			</return>
 			<description>
+				Returns the AABB wrapping by all existing particles at the current frame.
```

Comment:
Unclear, I think the grammar is off. 
Suggestion: Returns the AABB... bounding box? that encompasses/contains all the particles that are active in the current frame.

Reply:
Yes, sorry about that. I've corrected the line.